Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupThemeComboBox.cs

```diff
@@ -22,6 +22,10 @@ namespace Krypton.Ribbon
     [DefaultProperty(nameof(Text))]
     public class KryptonRibbonGroupThemeComboBox : KryptonRibbonGroupComboBox, IKryptonThemeSelectorBase
     {
+        /*
+         * TODO: grouped Ribbon controls do expose designers, needs a closer look
```

Comment:
If you use `// ToDo:` it will allow ReSharper to pick it up

Reply:
@Wagnerp 
I never used Resharper.
What will happen then?

Does it need to be removed?

Reply:
@giduac ReSharper has the ability to pick up `ToDo`s. Please DM me in Discord, as I have further information for you. But just replace the /**/ with //, and it'll be ok.

Reply:
@Wagnerp 
Updated